invisible_cities/cities/esmeralda_test.py

```diff
@@ -176,6 +186,7 @@ def test_esmeralda_all_hits_after_drop_voxels(esmeralda_config, Th228_hits, conf
 
 
 #TODO: refactor paolina to include this as a filter
+@ignore_warning.no_config_group
```

Comment:
similarly, this ignore_warning for no_config group is (I think) applied to every test in this file. Is it possible to make that a single line that just states "ignore this warning for all tests in `esmeralda_test.py`"?

Reply:
I don't know, there might be, but I am hesitant to do so for a similar reason. If someone in the future adds a test that is not supposed to raise a warning, but it does, the developer will not see it.